### PR TITLE
Added Virtual scroll Select component

### PR DESCRIPTION
**Pull request by bohdan-timechimp**

### CODE REVIEW

File: src/components/fixed-size-select/FixedSizeSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+      valueKey="label"
+      placeholder="Select"
+      overrides={{
+        Dropdown: { component: VirtualDropdown },
+        Root: {
+          style: {
+            width: 'auto',
+            height: '44px',
+            paddingTop: '6px',
+            paddingRight: '8px',
```
Comment (hiddeottens) [inline src/components/fixed-size-select/FixedSizeSelect.tsx:126]:
```suggestion
            ...padding(scale200, scale300, '0', '0')
```

File: src/components/fixed-size-select/FixedSizeSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+            backgroundColor: '#fff',
```
Comment (hiddeottens) [inline src/components/fixed-size-select/FixedSizeSelect.tsx:127]:
```suggestion
            backgroundColor: primaryB,
```

File: src/components/fixed-size-select/FixedSizeSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+          },
+        },
+        ControlContainer: {
+          style: {
+            height: '38px',
```
Comment (hiddeottens) [inline src/components/fixed-size-select/FixedSizeSelect.tsx:132]:
```suggestion
            height: scale975,
```

File: src/components/fixed-size-select/FixedSizeSelect.tsx [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+            backgroundColor: '#fff',
```
Comment (hiddeottens) [inline src/components/fixed-size-select/FixedSizeSelect.tsx:133]:
```suggestion
            backgroundColor: primaryB
```

File: src/components/fixed-size-select/FixedSizeSelect.tsx [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+            }),
+            ...borderRadius(radius200),
+            ...padding('0'),
+          },
+        },
+        ValueContainer: {
+          style: {
+            ...padding('0'),
+            paddingTop: '6px',
+            paddingLeft: '10px',
```
Comment (hiddeottens) [inline src/components/fixed-size-select/FixedSizeSelect.tsx:146]:
```suggestion
            ...padding(scale200, '0', '0', scale400),
```

**Review by hiddeottens — COMMENTED** [review]
Looking good! just some feedback for the `theme` stuff

File: src/components/fixed-size-select/FixedSizeSelect.tsx [hunk 6]
```diff
@@ -0,0 +1,154 @@
+import * as React from 'react';
+import { withStyle } from 'baseui';
+import { Option, Select, StyledDropdownListItem, Value } from 'baseui/select';
+import { StyledList, StyledEmptyState, OptionListProps } from 'baseui/menu';
+import { FixedSizeList } from 'react-window';
+import { border, borderRadius, padding } from 'utils';
```
Comment (hiddeottens) [inline src/components/fixed-size-select/FixedSizeSelect.tsx:6]:
```suggestion
import { border, borderRadius, padding } from '../../utils';
```

File: src/components/fixed-size-select/FixedSizeSelect.tsx [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+import { useTheme } from 'providers';
```
Comment (hiddeottens) [inline src/components/fixed-size-select/FixedSizeSelect.tsx:7]:
```suggestion
import { border, borderRadius, padding } from '../../utils';
import { useTheme } from '../../providers';
```

File: src/components/fixed-size-select/FixedSizeSelect.tsx [hunk 8]
```diff
@@ -0,0 +1,154 @@
+import * as React from 'react';
+import { withStyle } from 'baseui';
```
Comment (hiddeottens) [inline src/components/fixed-size-select/FixedSizeSelect.tsx:2]:
```suggestion
import { themedWithStyle } from '../../theme';
```

File: src/components/fixed-size-select/FixedSizeSelect.tsx [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+import { useTheme } from 'providers';
+import { FixedSizeSelectProps } from './types';
+
+const LIST_ITEM_HEIGHT = 42;
+const EMPTY_LIST_HEIGHT = 72;
+const MAX_LIST_HEIGHT = 200;
+
+const ListItem = withStyle(StyledDropdownListItem, {
+  paddingTop: 0,
+  paddingBottom: 0,
```
Comment (hiddeottens) [inline src/components/fixed-size-select/FixedSizeSelect.tsx:16]:
```suggestion
  ...padding('0')
```

File: src/components/fixed-size-select/FixedSizeSelect.tsx [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+        sizing: { scale0, scale200, scale300, scale400, scale500, scale600, scale800 },
+        borders: { border100, border300, radius200 },
+        colors,
+        customColors,
+        customSizing: { scale50, scale975 },
+      },
+    },
+  } = useTheme();
+  const { primaryB, borderTransparent, primary } = colors;
+  const { dark1, dark3, dark4, light2, light3, light4, light7 } = customColors;
```
Comment (hiddeottens) [inline src/components/fixed-size-select/FixedSizeSelect.tsx:111]: Screenshot 2023-06-21 at 09 37 55

Please remove all unused imports / props

**Review by danylo-hlibov-timechimp — APPROVED** [review]

**Pull request merged**